simplified-app-ekirjasto/src/main/assets/txnative/en/txstrings.json

```diff
@@ -159,6 +183,30 @@
       "accountWantChildCard" : {
          "string" : "Want a card for your child?"
       },
+      "account_about_app" : {
+         "string" : "About App"
+      },
+      "account_eula" : {
+         "string" : "User Agreement"
+      },
+      "account_leave_feedback" : {
+         "string" : "Leave feedback"
+      },
+      "account_licenses" : {
+         "string" : "Software Licenses"
+      },
+      "account_login_passkey" : {
+         "string" : "Sign in with Passkey"
+      },
+      "account_login_suomifi" : {
+         "string" : "Sign in with Suomi.FI"
```

Comment:
It should be "Suomi.fi", like it is in https://github.com/NatLibFi/ekirjasto-android-core/blob/c368f927032acd77962c4775182f325a6fed48f3/simplified-ui-accounts/src/main/res/values/strings.xml#L72

Reply:
These strings are just pulled from Transifex, so they should fixed there, otherwise they'll be overwritten by the next pull (and the pulls will later be automated on every merged PR).

Reply:
Ok for another specific PR.